renderer.php

```diff
@@ -826,8 +838,25 @@ private function construct_take_user_controls(attendance_take_data $takedata, $u
             }
 
             $celldata['text'] = array();
+
```

Comment:
there shouldn't be any hard-coded references to specific statuses like Present/Late/Excused/Absent - haven't looked closely at what this code is doing, but to get this into the core release we can't be hard-coding this stuff.

Reply:
i agree. this needs to be removed. will work on this.